ouroboros-consensus-cardano/changelog.d/20230612_125709_armandoifsantos_dynamic_block_forging.md

```diff
@@ -0,0 +1,24 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+
+### Non-Breaking
+
+- Refactored code due to the fact that block forging credentials got extracted out of
+  `ProtocolInfo` type.
+
+### Breaking
+
+- Change numerous functions return type to include block forging credentials since
+  they got extracted out of `ProtocolInfo` type.
+  - Refactored the type signatures to accommodate the fact that `ProtocolInfo` does not
```

Comment:
```suggestion
  - Refactor the type signatures to accommodate the fact that `ProtocolInfo` does not
```

Reply:
In general:

> Write your commit message in the imperative: "Fix bug" and not "Fixed bug"
or "Fixes bug."  This convention matches up with commit messages generated
by commands like git merge and git revert.

https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines